OpenRA.Mods.Common/Traits/Conditions/ConditionManager.cs

```diff
@@ -219,23 +187,12 @@ public int RevokeCondition(Actor self, int token)
 		}
 
 		/// <summary>Returns true if the given external condition will have an effect on this actor.</summary>
-		public bool AcceptsExternalCondition(Actor self, string condition, bool timed = false)
+		public bool AcceptsExternalCondition(Actor self, string condition)
 		{
 			if (state == null)
 				throw new InvalidOperationException("AcceptsExternalCondition cannot be queried before the actor has been fully created.");
 
-			if (!externalConditions.Contains(condition))
-				return false;
-
-			// A timed condition can always replace an existing timed condition (resetting its duration)
-			if (timed && timers.ContainsKey(condition))
```

Comment:
This was a workaround for #12498, which is no longer needed because replacing the boolean states with levels implies that whitelisted external conditions are *always* allowed.  I'll have some rather unpleasant words for anyone who suggests that we reintroduce explicit level caps for arbitrary conditions :wink: